packages/schemas/tables/sso_connectors.sql

```diff
@@ -0,0 +1,29 @@
+create table sso_connectors (
+  tenant_id varchar(21) not null 
+    references tenants (id) on update cascade on delete cascade,
+  /** The globally unique identifier of the SSO connector. */
+  id varchar(128) not null,
+  /** The connector factory name of the SSO provider. */
+  provider_name varchar(128) not null,
+  /** The name of the SSO provider for display. */
+  connector_name varchar(128) not null,
```

Comment:
nit: can simply use `name` or `displayName`

Reply:
aligned with the name on design